components/apimgt/org.wso2.carbon.apimgt.core/src/test/java/org/wso2/carbon/apimgt/core/impl/UserAwareAPIStoreTestCase.java

```diff
@@ -1,192 +0,0 @@
-/*
```

Comment:
After a discussion with Malintha, we realized the validation should be there for now. Will restore this class and the validation logic.